src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");

```

Comment:
The message in the `doWork` method should indicate what type of `Machine` is working. Consider specifying that it is an `Excavator` in the message to make it more informative.